### ISSUE TITLE
multiOTP Enterprise in 64 bits OVA appliance

### ISSUE THREAD

**Issue by karkaaa** [multiOTP/multiotp#65]
Hello,

I got email from @multiOTP.com with this information:

> 64 bits OVA appliance of multiOTP Enterprise in 5.6.x.x version will be available for download the 16. July 2019

What is the status of new release?

**Comment by multiOTP** [comment]
Hello "Karkaaa",

Due to some stability issues, we have waited some fixes in the new Debian Buster edition in order to release our new official OVA appliance. The new Debian version 10.1 has been released the 7. September 2019, and if all QA tests are going well with this version, the 64 bits VM release of multiOTP Enterprise (5.7.x.x) will be available the 18. September 2019, and multiOTP Pro and multiOTP open source will follow i nthe next days.

We are sorry for this delay, but we are sure that you will appreciate a stable package.

Best regards,
